tmt/steps/provision/mock.py

```diff
@@ -203,17 +235,17 @@ def enter_shell(self) -> None:
         self.parent.verbose('mock', 'Shell is ready.', color='blue', level=3)
 
         # We do not expect these commands to fail.
-        self.mock_shell.stdin.write('rm -rf /srv/tmt-mock\n')
-        self.mock_shell.stdin.write('mkdir /srv/tmt-mock\n')
+        self.mock_shell.stdin.write(f'rm -rf /{MOCK_PIPE_STEM}\n')
+        self.mock_shell.stdin.write(f'mkdir /{MOCK_PIPE_STEM}\n')
         self.mock_shell.stdin.write(
             'mkfifo'
-            ' /srv/tmt-mock/stdout'
-            ' /srv/tmt-mock/stderr'
-            ' /srv/tmt-mock/returncode'
-            ' /srv/tmt-mock/filesync'
+            f' /{MOCK_PIPE_STDOUT_STEM}'
+            f' /{MOCK_PIPE_STDERR_STEM}'
+            f' /{MOCK_PIPE_RETURNCODE_STEM}'
+            f' /{MOCK_PIPE_FILESYNC_STEM}'
             '\n'
         )
-        self.mock_shell.stdin.write('chmod a+rw /srv/tmt-mock/filesync\n')
+        self.mock_shell.stdin.write(f'chmod -R a+rw /{MOCK_PIPE_STEM}\n')
         self.mock_shell.stdin.flush()
```

Comment:
Fine to deffer, but do you think you could generalize these, similar to how `ShellScript` is handled (or maybe reuse that completely)? It is unintuitive how the stdin translates to command execution.

Reply:
> Maybe I can put more comments on it?

Yes please :slightly_smiling_face: